city_scrapers/spiders/chi_ssa_4.py

```diff
@@ -0,0 +1,172 @@
+import json
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa4Spider(CityScrapersSpider):
+    name = "chi_ssa_4"
+    agency = "Chicago Special Service Area #4 South Western Avenue"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://95thstreetba.org/events/category/board-meeting/?"
+        "tribe_paged=1&tribe_event_display=list&tribe-bar-date=2018-10-01"
+    ]
+
+    def parse(self, response):
```

Comment:
This might be better in a `start_requests` method actually (as opposed to my comment above). Here's an example of that:

https://github.com/City-Bureau/city-scrapers/blob/20a12ba5d76186cba65b45f7f764f02393d4a991/city_scrapers/spiders/cook_county.py#L17-L27

Reply:
See my above comment.

I think I'll have it look just one month in the future then.

I'll resolve both these comments when it's implemented and request another review

Reply:
Follow up to the template of looping through the dates. I tested this out and this only returns two dates (today and next_month) instead of all dates in between. I'm not sure if that is expected behavior. The implementation below iterates through a start and end date. This is for reference in case someone is looking for this.

```suggestion
   from datetime import datetime, timedelta
   
   '''Next 90 days'''
    today = datetime.now()
    tomorrow = today.replace(day=today.day+1)
    next_months = tomorrow + timedelta(days=90)
    delta = timedelta(days=1)
    while today <= next_months:
       print(today.strftime("%Y-%m-%d"))
       today += delta    
```